src/net_processing.cpp

```diff
@@ -3647,6 +3653,21 @@ bool PeerLogicValidation::SendMessages(CNode* pto, std::atomic<bool>& interruptM
     return true;
 }
 
+void setAutoRequestBlocks(bool state)
+{
+    fAutoRequestBlocks = state;
+}
+
+bool isAutoRequestingBlocks()
+{
+    return fAutoRequestBlocks;
+}
+
+unsigned int getAmountOfBlocksInFlight() {
```

Comment:
Brace starts on same line for this function but on different lines for above two functions. I think separate line is the preferred style these days, but I could be wrong?